edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/Startup.cs

```diff
@@ -47,6 +48,12 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
 
         public void Configure(IApplicationBuilder app)
         {
+            // Seperate metrics endpoint from https authentication
+            app.Map("/metrics", metricsApp =>
```

Comment:
Does this mean we still allow other paths on 9600 port (like Twin, etc?)

Reply:
I think it might. Is that an issue? It will still require the same authentication

Reply:
If I am understanding this correctly, ASP.NET is listening on two ports and we are concerned about duplicating the existing endpoints (i.e. twin) on this new metrics port? I feel like we shouldn't duplicate even if it requires the same authentication.

Reply:
Technically, you use MapWhen to filter out requests with path other than "/metrics" on 9600 port.  But this is same as before since request of any path can send to edgeHub.

Reply:
If we can filter out non metrics requests on 9600 port, then why not do so? Before we were only listening on the main port (and technically 443). Now we are introducing another with the customer expectation as only-used-for-metrics. 

Also, I don't see how https should work considering the UseHttps() logic in Hosting.cs. Even if this works, it would be nice to eliminate the confusion by restricting the paths. I'm guessing I don't have full context here though.